client/src/components/cityview/realm/villagers/panels/townhall/NpcChat.tsx

```diff
@@ -28,14 +28,19 @@ const NpcChat = ({}) => {
   const bottomRef = useRef<HTMLDivElement>(null);
 
   useEffect(() => {
-    if (selectedTownhall === null) {
+    if (selectedTownhall === null || localStorage.getItem(LOCAL_STORAGE_ID) == null) {
+      console.log(selectedTownhall);
```

Comment:
delete console.log

Reply:
I kept this intentionally, not sure if my fix is really fixing the problem so I want to log the `selectedTownhall` there in case the bug happens again and I can see if it's in that condition that it fails. Is that ok for you?